### DISCUSSION TITLE
Equilibrage combat

### DISCUSSION THREAD

**Discussion by BananePlantain** [Crownicles/Crownicles#2218] [Nouvelles idées]
![série](https://github.com/DraftBot-A-Discord-Adventure/DraftBot/assets/126412670/1965f55c-34bb-48f0-983f-1741d3ed3766)

**Comment by Procion06** [comment]
l'idée est bonne sur le papier après j'ai peur qu'il y ait trop d'arrangements et/ou au contraire que ça bloque certains fights car les joueurs préfèreront attendre d'être sûr de pouvoir gagner avant de prendre un combat
en gros je pense que ça va fausser le game
par contre quand il y aura les combats contre l'ia, ça pourrait être top

**Comment by Easygamer01s** [comment]
Peut-on imaginer une mission dans laquelle on doit gagner des combats plutôt qu'en faire simplement ? Avec un temps très court pour la réaliser ça devrait donner un effet similaire..
Après pour les arrangements entre joueurs c'est clairement inhérents au jeu..
